.github/workflows/check_compile.yml

```
@@ -62,7 +62,7 @@ jobs:
             setup-backend: |
               wget -r -np -nd -A 'amdgpu*.deb' https://repo.radeon.com/amdgpu-install/latest/ubuntu/noble/
               sudo apt install ./amdgpu-install_*.deb
-              sudo apt-get update && sudo apt-get install -y amdgpu-dkms rocm-dev
+              sudo apt-get update && sudo apt-get install -y amdgpu-dkms rocm-dev hipsolver-dev hipblas-dev
               ROCM_DIR=$(ls -d /opt/rocm-*)
               echo "backend-path=$ROCM_DIR" >> $GITHUB_OUTPUT
```

Set the cmake prefix here if you need it to be part of the ci. Not in the cmake file.
```suggestion
              sudo apt-get update && sudo apt-get install -y amdgpu-dkms rocm-dev hipsolver-dev hipblas-dev
              ROCM_DIR=$(ls -d /opt/rocm-*)
              echo "CMAKE_PREFIX_PATH=CMAKE_PREFIX_PATH:$ROCM_DIR" >> $GITHUB_ENV
              echo "backend-path=$ROCM_DIR" >> $GITHUB_OUTPUT
```